### DISCUSSION TITLE
Some Epic games do not work

### DISCUSSION THREAD

**Discussion by Anubis1971** [moraroy/NonSteamLaunchers-On-Steam-Deck#263] [General]
I’ve tried a few games that simply do not launch (i.e. Bioshock Infinite) but work on Heroic.  I prefer to use NSL but would like to know if there is a way to fix games that do not launch or can determine which games will or will not work prior to installation.  Know this would save hundreds of GBs of download bandwidth.

**Comment by moraroy** [comment]
@Anubis1971 protondb.com

**Reply by Anubis1971** [reply]
I meant which games will work when installed through NSL not just on SD in general.  Some do not work just not sure why.  The same game(s) works on Heroic.  Would really prefer to use one type of launcher.

**Reply by moraroy** [reply]
@Anubis1971 some games require extra steps to get running, I also have bioshock infinite, and report back to you. If you prefer one type of launcher I would try lutris or heroic, every game is different. There is some bugs in nsl that I still have to fix, but the shortcut it creates for bioshock should work, I'll test

**Reply by Anubis1971** [reply]
@moraroy Thank you for checking it out!  Perhaps knowing exactly what extra steps you mention are, would help us try them out for each game which do not work.  

Normally i wouldn’t care.  If one launcher doesn’t work, I’ll try another.  However, I have a metered connection and games are getting huge, not to mention it takes a long time to download and install on the SD :)

**Reply by moraroy** [reply]
@Anubis1971 yeah I googled it and found out it's an epic launcher issue, I can't launch it either. You may have to do some research as to why lutris or heroic can launch it, I'll do some testing, supposedly bioshock is supposed to open up the "2K Launcher" or something like that

**Reply by moraroy** [reply]
@Anubis1971 check out this YouTube video, https://youtu.be/2Nj51woakpY?si=t6JENKfNF9vT73gn I can do one of two things, I can add the 2K Launcher to NSL. Or you can edit that file. If you find a better way that can help everyone as whole let me know.

**Reply by moraroy** [reply]
@Anubis1971 I fixed it on mine, so basically what I did was I made sure my epic games account was linked, go to the 2K website and link it if you want, then once you have a 2k and an epic account linked, I did what the guy did in the video, and edited that file. Once I did that, I then went back into game mode and was able to access the game via the Shorcut that NSL provided for you for bioshock Inifinite.

**Reply by moraroy** [reply]
@Anubis1971 Here is my steam deck notice it says account linked and it seems I also have cloud saves
